proposals/tsc-voting-procedures.md

```diff
@@ -0,0 +1,199 @@
+# TSC Voting Procedures Proposal
+
+## Overview
+
+This proposal establishes standardized voting procedures for the Hiero Technical Steering Committee (TSC) to improve decision-making efficiency while maintaining transparency and adherence to charter requirements.
+
+## Voting Methods
+
+### 1. In-Meeting Votes
+- **When Used**: For decisions during TSC meetings with quorum present
+- **Threshold**: Simple majority of members present
+- **Recording**: Documented in meeting minutes with vote counts
+
+### 2. Asynchronous Votes via GitHub Pull Request
+- **When Used**: 
+  - When quorum cannot be achieved in meetings
+  - For decisions requiring extended review/discussion
+  - For charter-mandated votes (e.g., license exceptions)
+- **Location**: Pull requests in the `proposals/` directory
+- **Notification**: All TSC members must be notified via:
+  - GitHub PR mention (@username)
+  - Slack notification in TSC channel
+
+## Voting Process
+
+### Step 1: Discussion (Optional)
+- Create a GitHub Issue for complex proposals requiring discussion
+- Label with "tsc-discussion"
+- No formal timeline - continue until consensus on proposal language
+
+### Step 2: Proposal Creation
+1. Create a new file in `proposals/` directory following the template
+2. File naming convention: `YYYY-MM-DD-brief-description.md`
+3. Open a Pull Request with the finalized proposal
+4. **Proposal is immutable once PR is opened** - no changes allowed
+5. Link to discussion issue if one exists
+
+### Step 3: Voting Period
+- **Duration**: 7 calendar days maximum
```

Comment:
I disagree with that. People can be on vacation and not able to vote. Since we have "Mathematically impossible to change outcome" we should extend this periode